src/components/inputs/LocationType.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div ref="rootEl">
+    <button :class="selectedClass" @click="onTriggerClick"></button>
+    <transition name="fade">
+      <ul v-if="isListVisible">
+        <li v-for="option in options" v-bind:key="option.value">
+          <a :class="option.key" alt="option.description" @click="onListItemClick(option.value)">{{
```

Comment:
Should an anchor tag have an `alt` attribute?

Reply:
You are right, it was title that I was looking for. Good spot!